tests/validators/test_dict.py

```diff
@@ -21,24 +22,29 @@ def test_dict(py_and_json: PyAndJson):
         v.validate_test([])
 
 
+@dataclass
+class Foobar:
+    x = 1
```

Comment:
This is just because if you create a class dynamically and use `repr` as the ids like we do in the test below you end up with issues because the repr (and hence the test name) changes every time pytest discovers tests (because the id is different).

Reply:
`pytest.param` allows setting test ids explicitly. Also, using those ids to give an explicit meaning to test cases could help with reading the code later.

Reply:
Good call I’ll do that instead